### PR TITLE
Fixes #13592 - Move override views under app/views

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: lib/katello/engine.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module Katello
   class Engine < ::Rails::Engine
     isolate_namespace Katello
+    paths['app/views'] << "app/overrides/foreman"
```
Comment (daviddavis) [inline lib/katello/engine.rb]: I tried to encapsulate this into an initializer but it didn't seem to work.

**Comment by ehelms** [comment]
Would I be correct in asserting that this issue should plague any plugin that is using deface?

**Comment by daviddavis** [comment]
@ehelms Depends I think on whether they are using a different directory other than app/views.

**Comment by daviddavis** [comment]
I moved the templates to `app/views/katello/overrides` but let me know if you think they should go elsewhere.

**Comment by ehelms** [comment]
Would app/views/overrides or app/views/foreman overrides work so they stand out or e related to what they override work?

**Comment by daviddavis** [comment]
[test]

**Comment by daviddavis** [comment]
I moved the views to app/views/overrides.

**Comment by ehelms** [comment]
ACK

**Pull request merged**